portingdb/htmlreport.py

```diff
@@ -729,6 +747,49 @@ def mispackaged():
         mispackaged=mispackaged,
     )
 
+
+def namingpolicy():
+    """Naming policy tracking.
+    """
+    db = current_app.config['DB']()
+    query = db.query(tables.Package)
+    total_packages = query.count()
+    query = query.join(tables.CollectionPackage)
+    query = query.filter(tables.CollectionPackage.collection_ident == 'fedora')
+    query = query.filter(tables.CollectionPackage.is_misnamed.is_(True))
+    total_misnamed = query.count()
+    total_correct = total_packages - total_misnamed
+
+    # Progress bar info.
+    progress = get_naming_progress_info(total_correct, total_misnamed)
+
+    # Statuses with number of packages
+    statuses = OrderedDict(
+        db.query(tables.Status, func.count(tables.Package.name))
+        .outerjoin(tables.Status.packages)
+        .join(tables.CollectionPackage)
+        .filter(tables.CollectionPackage.collection_ident == 'fedora')
+        .filter(tables.CollectionPackage.is_misnamed.is_(True))
+        .group_by(tables.Status.ident)
+        .order_by(tables.Status.order))
```

Comment:
In the beginning of this project, I made a normalized relational database. But as the project grew, I realized it's only edited by dropping everything and re-loading from the data in Git, so normalization doesn't really make sense.

Please add a column to the database if it makes your job easier. The DB is effectively a cache.
That said no reason to add columns until needed. If you already wrote code without the column, keep it.

As for the review, could you ping me once @hroncok's comments are addressed?